### ISSUE TITLE
Highlight circular dependencies in graph

### ISSUE THREAD

**Issue by timoxley** [antfu-collective/vite-plugin-inspect#121]
### Clear and concise description of the problem

Sometimes I get errors that appear due to (indirect) circular dependencies. It would be great if I could visualize these in the inspector.

### Suggested solution

Use different colours to highlight indirect and direct circular dependencies in the inspector graph when a node is selected.

### Alternative

Log circular dependency chains in the output somewhere.

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by antfu** [comment]
That would be great, would you be interested in helping out?

**Comment by timoxley** [comment]
> That would be great, would you be interested in helping out?

yes, I did have a poke around to see if it was trivial, but I didn't see an immediately obvious way to plug this in. How would you make this change?
